test/mode/modeNormal.test.ts

```diff
@@ -1153,7 +1153,7 @@ suite("Mode Normal", () => {
       title: "Can do S",
       start: ["    one", "    tw|o", "    three"],
       keysPressed: "2S",
-      end: ["    one", "|"]
+      end: ["    one", "        |"]
```

Comment:
This is the weirdness mentioned in the PR. I don't know enough about `testSimplifier` to understand why the behavior is different there.